public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+        the next instruction was located.
+
+
+Thus, the bit layout of the instruction needs an additional component for the next address `(N)`. Its location doesn't affect the hack, as described in the story, but let's place it on the least significant bits, to be on the safe side: 
+
+            AAAXCCCNNN
+        MSB <----------> LSB
+   
+    
+We know that the `(A)` bits are lower than the `(C)` bits, because Nather later describes the overflow that Mel hijacked:
+
+        Instead, he would pull the instruction into a machine register,
+        add one to its address,
+        and store it back
+        ...
+        He had located the data he was working on
+        near the top of memory —
+        the largest locations the instructions could address —
+        so, after the last datum was handled,
+        incrementing the instruction address
+        would make it overflow.
+        The carry would add one to the
+        operation code, changing it to the next one in the instruction set:
+
+
+If incrementing the address span overflowed into the opcode span, then the bit order between them is established:
+
+            CCCXAAANNN
+        MSB <----------> LSB
+
+If the index register bit `(X)` is indeed between the two and turned on, then overflowing the `(A)` span will carry through `(X)` into the `(C)` span, incrementing it by one. The result:
+
+        a jump instruction.
+        Sure enough, the next program instruction was
+        in address location zero,
+        and the program went happily on its way.
+
+### The Unpleasant Truth
+
+All this is possible in theory, but we still don't know where the `JUMP` instruction takes its operand from. We know that the operand's value must be `0` and it should be ready. The address span `(A)` indeed contains `0`, but nowhere in the description of RPC-4000's architecture, does Nather mention an option of the data address field doubling as an instruction address. The `JUMP` instruction could take its value from some register, but that `0` would have to be stored there beforehand, providing a screaming clue that some operation was being set up.
+
+Not only did I gloss over this crucial step for 30 years - I also ignored the absence from this part of the story, of the unique addressing mode described above. The one that included the next instruction address in every instruction(!). Was that part of affected in the overflow? Was there any relationship between the `JUMP` opcode and the `(A)` or `(N)` fields? And what about that `(X)` bit? Its alleged location, between two bitfields, just seemed... wrong.
+
+After demoting myself from 'computer guy' to 'A guy who likes to think of himself as such', I took the basic step required to understand Mel's hack: looking up the [RPC-4000 manual](http://www.bitsavers.org/pdf/royalPrecision/RPC-4000/RPC-4000_Programming_Manual.pdf). It didn't take much browsing to hit a figure that dispelled all my doubts.
```

Comment:
The more I think about this joke, the less I like it. Why the self humbling? Sarcasm may not read correctly here. Try to admit your realization in a more positive way.